projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -341,15 +341,15 @@ public function add_plugins_menu( $wp_admin = false ) {
 	 */
 	public function add_users_menu( $wp_admin = false ) {
 		if ( current_user_can( 'list_users' ) ) {
+			// We shall add the Calypso user management & add new user screens at all cases ( Calypso & Atomic ).
+			$submenus_to_update = array(
+				'user-new.php' => 'https://wordpress.com/people/new/' . $this->domain,
+				'users.php'    => 'https://wordpress.com/people/team/' . $this->domain,
+			);
 			if ( ! $wp_admin ) {
-				$submenus_to_update = array(
-					'users.php'    => 'https://wordpress.com/people/team/' . $this->domain,
-					'user-new.php' => 'https://wordpress.com/people/new/' . $this->domain,
-					'profile.php'  => 'https://wordpress.com/me',
-				);
-				$this->update_submenus( 'users.php', $submenus_to_update );
+				$submenus_to_update = array_merge( $submenus_to_update, array( 'profile.php' => 'https://wordpress.com/me' ) );
```

Comment:
Nitpick: Since we're only adding one more item, we can avoid using `array_merge`.

```suggestion
				$submenus_to_update['profile.php'] = 'https://wordpress.com/me';
```

Reply:
Sure! 7b96b89